presto-docs/src/main/sphinx/presto-cpp.rst

```diff
@@ -45,8 +45,11 @@ Only specific connectors are supported in the Presto C++ evaluation engine.
 
 * Hive connector for reads and writes, including CTAS, are supported.
 
+* Hive connector supports Delta table access using Symlink table using Hive Connector in Prestissimo.
+  Details about Delta Symlink table is available `here <https://docs.delta.io/latest/presto-integration.html>`_.
```

Comment:
```suggestion
  For more information about Symlink tables, see `Presto, Trino, and Athena to Delta Lake integration using manifests <https://docs.delta.io/latest/presto-integration.html>`_.
```
When creating links, it is better to provide words that help the reader know where they are going to go before they click the link. Instead of "here" or similar phrasing, an easy way to do this is to use the title of the destination page.